tests/test_database.py

```diff
@@ -80,3 +85,26 @@ def test_current_timestamp(self):
         db = get_conn(self.db_cls)
         res = db.query(current_timestamp(), datetime)
         assert isinstance(res, datetime), (res, type(res))
+
+@test_three_part_dbs
+class TestThreePartIds(unittest.TestCase):
```

Comment:
@erezsh is this kinda what you had in mind?

Not sure if there's a better way to get the current_database/current_schema

Reply:
And I think BQ does not support current_database() so will need to handle differently